samples/apps/copilot-chat-app/webapp/src/components/chat/ChatResourceList.tsx

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import {
+    Avatar,
+    PresenceBadgeStatus,
+    Table,
+    TableBody,
+    TableCell,
+    TableCellLayout,
+    TableHeader,
+    TableHeaderCell,
+    TableRow,
+    makeStyles,
+    shorthands,
+    tokens,
+} from '@fluentui/react-components';
+import { DocumentPdfRegular, DocumentTextRegular } from '@fluentui/react-icons';
+import * as React from 'react';
+import { ChatMemorySource } from '../../libs/models/ChatMemorySource';
+import { useChat } from '../../libs/useChat';
+
+const useClasses = makeStyles({
+    root: {
+        ...shorthands.margin(tokens.spacingVerticalM, tokens.spacingHorizontalM),
+    },
+    table: {
+        backgroundColor: tokens.colorNeutralBackground1,
+    },
+    tableHeader: {
+        fontWeight: '600',
+    },
+});
+
+interface ChatResourceListProps {
+    chatId: string;
+}
+
+export const ChatResourceList: React.FC<ChatResourceListProps> = ({ chatId }) => {
+    const classes = useClasses();
+    const chat = useChat();
+    const [resources, setResources] = React.useState<ChatMemorySource[]>([]);
+
+    React.useEffect(() => {
+        chat.getChatMemorySources(chatId).then((sources) => {
+            setResources(sources);
+        });
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [chatId]);
+
+    const items = resources.map((item) => ({
+        name: {
+            label: item.name,
+            icon: getFileIconByFileExtension(item.name),
+            url: item.hyperlink,
+        },
+        updatedOn: {
+            label: new Date(item.updatedOn).toLocaleTimeString([], {
```

Comment:
If the timestamp isn't today, only show the date instead of time

where date = new Date(item.updatedOn);

```suggestion
            label: date.toDateString() !== new Date().toDateString() ? date.toLocaleDateString([], {
            month: 'numeric',
            day: 'numeric',
        }) : date.toLocaleTimeString([], {
        hour: '2-digit',
        minute: '2-digit',
    })
```

Reply:
done. created a helper fn to help us render this across the app